pages/en/lb4/Using-components.md

```diff
@@ -7,24 +7,42 @@ sidebar: lb4_sidebar
 permalink: /doc/en/lb4/Using-components.html
 summary:
 ---
-Components are an important part of LoopBack 4. We are keeping the core small and easy to extend, making it easy for independent developers to contribute additional features to LoopBack. Components are the vehicle for bringing more goodness to your applications.
 
-A typical LoopBack component is an [npm](https://www.npmjs.com/) package exporting a Component class which can be added to your application.
+One of the many ways LoopBack 4 allows for extensibility is through Components.
+A Component makes it easy for independent developers to contribute additional
+features to LoopBack. Components provide a package that allows you to extend
+your Application.
```

Comment:
I see `Component` as the mechanism to bundle and group extension contributions. Component is part of the extensibility story for LoopBack 4 but it's not one of the many ways.

BTW, a package (npm module) can possibly export multiple components in theory.

Reply:
Yep. A package can export multiple components. I think ideally one package should export only one Component. That said, we can't control extensions and extension developers should show users how to consume the extension. (Component vs. Mixin vs. manually adding artifacts).